### DISCUSSION TITLE
deceptive site ahead prbolem in vercel

### DISCUSSION THREAD

**Discussion by reshad21** [vercel/community#3527] [Help] [answered]
### Summary

deceptive site ahead prbolem in vercel

### Example

content-management-application.vercel.app

### Steps to Reproduce

deceptive site ahead prbolem in vercel

**Comment by swarnava** [comment, marked as answer]
Hi,

This sounds to me like the Chrome deceptive site warning, which is associated with the domain rather than a Vercel error. Information on this can be found here: https://developers.google.com/search/docs/monitor-debug/security/social-engineering

You need to contact Google team to unblock this: https://safebrowsing.google.com/safebrowsing/report_error/?url=http%3A%2F%2Fcontent-management-application.vercel.app%2F&hl=en-GB

Let us know if you require any further assistance and we will be happy to help
